test/unit/web/rtc_videw_view_test.dart

```diff
@@ -1,12 +1,12 @@
 @TestOn('browser')
 import 'package:flutter_test/flutter_test.dart';
-import 'package:flutter_webrtc/flutter_webrtc.dart';
+import 'package:flutter_webrtc/flutter_webrtc.dart' hide RTCVideoRenderer;
```

Comment:
I don’t quite understand why `hide RTCVideoRenderer` here

Reply:
This an error.  But this test is not working anyway.  I will write a working test when the below issue will be resolve.
The annotation @TestOn is not working.  see https://github.com/flutter/flutter/issues/66045